src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
+.UserInfo {
```

Comment:
Classes are called with a lowercase letter

Reply:
In this task don't change classes in the given markup (tests use them);

Reply:
Accepted